libs/libxx/libcxx.defs

```diff
@@ -121,24 +115,6 @@ endif
 #          ~~~~~~~~~~^~~~~~~~~~~~
 libcxx/src/condition_variable.cpp_CXXFLAGS += -Wno-sign-compare
 
-# The following warning was observed with sim:libcxxtest compiled with
```

Comment:
What about other errors described above? They are not fixed with the new version, right?

Reply:
I will try to remove them later.